### PR TITLE
feat: add sendCapacity interface in cell module

### PR DESCRIPTION
**Pull request by classicalliu**

### CODE REVIEW

**Comment by ashchan** [comment]
@classicalliu CI fails.

**Comment by classicalliu** [comment]
@ashchan CI passed

File: packages/neuron-wallet/src/transaction.ts [hunk 1]
```diff
@@ -9,19 +10,121 @@ export interface Transaction {
...
   return {
     totalCount: page * perPage,
     transactions: Array.from({
       length: perPage,
     }).map(() => transaction),
-    addresses,
   }
 }
 
+const verifyScriptJsonObject = () => {
```
Comment (Keith-CY) [inline packages/neuron-wallet/src/transaction.ts]: This name comes with ruby-sdk, but not proper in JS, we can just name it `verifyScript`. Let's update it in the future.

File: packages/neuron-wallet/src/transaction.ts [hunk 2]
```diff
@@ -9,19 +10,121 @@ export interface Transaction {
...
+  return {
+    version: 0,
+    reference: systemScriptCellHash,
+    signedArgs,
+  }
+}
+
+// it's still a mocked interface
+// FIXME: replace number with BigInt
+const gatherInputs = async (capacity: string, minCapacity: string) => {
```
Comment (Keith-CY) [inline packages/neuron-wallet/src/transaction.ts]: This method has overlapped with sdk

File: packages/neuron-wallet/src/transaction.ts [hunk 3]
```diff
@@ -9,19 +10,121 @@ export interface Transaction {
...
+  if (inputCapacities < capacityInt) {
+    throw new Error('Capacity not enough!')
+  }
+  return {
+    inputs,
+    capacities: inputCapacities.toString(),
+  }
+}
+
+export const generateTx = async (address: string, capacity: string) => {
```
Comment (Keith-CY) [inline packages/neuron-wallet/src/transaction.ts]: This method has overlapped with sdk

File: packages/neuron-wallet/src/transaction.ts [hunk 4]
```diff
@@ -9,19 +10,121 @@ export interface Transaction {
...
+    version: 0,
+    deps: [systemScriptOutPoint],
+    inputs,
+    outputs,
+  }
+}
+
+// mock interface, use asw as ours wallet as mocked data.
+// TODO: should replace asw with wallet, now SDK data not be compatible
+export const sendCapacity = async (targets: { address: string; capacity: string }[]) => {
```
Comment (Keith-CY) [inline packages/neuron-wallet/src/transaction.ts]: This method has overlapped with sdk

Comment (classicalliu) [reply packages/neuron-wallet/src/transaction.ts]: Those methods should implement in wallet, not sdk, all data infos are in wallet

Comment (classicalliu) [reply packages/neuron-wallet/src/transaction.ts]: It's mocked methods, focus on interface

Comment (ashchan) [reply packages/neuron-wallet/src/transaction.ts]: I agree that we should pull out UTXO and tx generation logic out of SDK.

**Pull request merged**